src/apps/basic/basic_apps.lua

```diff
@@ -21,10 +22,20 @@ function Source:new(size)
    return setmetatable({size=size, packet=p}, {__index=Source})
 end
 
+txSeqNo= 0
+
 function Source:pull ()
    for _, o in ipairs(self.output) do
       for i = 1, link.nwritable(o) do
-         transmit(o, packet.clone(self.packet))
+         --transmit(o, packet.clone(self.packet))
+         local p= packet.clone(self.packet)
+         p.data[0]= txSeqNo %256
+         p.data[1]= (txSeqNo /2^8) %256
+         p.data[2]= (txSeqNo /2^16) %256
+         p.data[3]= (txSeqNo /2^24) %256
+         --p.data[0]= struct.pack("l", txSeqNo)
+         txSeqNo= txSeqNo +1
+         transmit(o, p)
```

Comment:
@eugeneia I have a few concerns about this code:

- Feature creep on `Source`. Could be a special-purpose inline app instead? (And refactor into `Source` if and when other apps need the same functionality? and even then perhaps make it optional rather than changing standard behavior?)
- The variable `txSeqNo` is global to the module so the sequence numbers from a given app will be non-deterministic and have discontinuities depending on how many instances are running. Better to make it a local instance variable of the app?
- Generally this change reduces consistency with the surrounding code (introducing camelCase and different use of whitespace around operators).